Multisig/UI/App/UIKit/TransactionDetailsViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  TransactionDetailsViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 23.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+class TransactionDetailsViewController: LoadableViewController, UITableViewDataSource, UITableViewDelegate {
+    var clientGatewayService = App.shared.clientGatewayService
+
+    private var cells: [UITableViewCell] = []
+    private var transaction: TransactionViewModel?
+    private var reloadDataTask: URLSessionTask?
+
+    private enum TransactionSource {
+        case id(String)
+        case safeTxHash(Data)
+        case data(TransactionViewModel)
+    }
+
+    private var txSource: TransactionSource!
+
+    // disable reacting to change of safes reactsToSelectedSafeChanges
+    override var reactsToSelectedSafeChanges: Bool { false }
```

Comment:
why? What is the use case?

Reply:
because I think that transaction details vc should not reload when safe changes, it should reload when transaction changes